src/actions/saved-events.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import type { Dispatch } from "redux";
+
+type AddSavedEventAction = { type: "ADD_SAVED_EVENT", id: string };
+type RemoveSavedEventAction = { type: "REMOVE_SAVED_EVENT", id: string };
+
+export type SaveEventAction = AddSavedEventAction | RemoveSavedEventAction;
```

Comment:
Should this be named `SavedEventAction` (past tense) for consistency?

Reply:
I just noticed, that we use a `StandardAction` type in the other actions. Does it make sense to use it here as well? I don't have strong feelings about this, though. Just nitpicking 🙂

Reply:
Cool. Makes sense to me.